javascript/hints.js

```diff
@@ -111,8 +111,8 @@ var titles = {
     "Resize width to": "Resizes image to this width. If 0, width is inferred from either of two nearby sliders.",
     "Resize height to": "Resizes image to this height. If 0, height is inferred from either of two nearby sliders.",
     "Multiplier for extra networks": "When adding extra network such as Hypernetwork or Lora to prompt, use this multiplier for it.",
-    "Discard weights with matching name": "Regular expression; if weights's name matches it, the weights is not written to the resulting checkpoint. Use ^model_ema to discard EMA weights.",
-    "Extra networks tab order": "Comma-separated list of tab names; tabs listed here will appear in the extra networks UI first and in order lsited.",
+    "Discard weights with matching name": "Regular expression; if weights name matches it, the weights is not written to the resulting checkpoint. Use ^model_ema to discard EMA weights.",
```

Comment:
`weights's`: not sure but I feel like the original is correct

Reply:
Thank you very much for your valuable suggestion. I will make the necessary modifications to the code as soon as possible

Reply:
this one was more of a discussion
as I'm not sure